app/derivative_services/default_derivative_service.rb

```diff
@@ -31,6 +31,7 @@ def change_set
   end
 
   def valid?
+    return false unless original_file
```

Comment:
What was happening that you needed to add all these?

Reply:
There's no `original_file`, so it delegates `mime_type` to original_file and then `.first` can't be called on nil.

Reply:
right, of course. and i was making use of that fact for the rest of my morning 😀  thanks